### PR TITLE
IGAPP-70: Implement test for internal linking

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: web/src/modules/common/components/__tests__/RemoteContent.spec.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 // @flow
 
 import React from 'react'
-import { shallow } from 'enzyme'
+import { mount, shallow } from 'enzyme'
```
Comment (klinzo) [inline web/src/modules/common/components/__tests__/RemoteContent.spec.js:4]: Could you solve this with the testing library or test renderer as well? Preferably the first?

**Review by klinzo — APPROVED** [review]

**Review by klinzo — APPROVED** [review]

File: native/android/.idea/misc.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <component name="FrameworkDetectionExcludesConfiguration">
     <file type="web" url="file://$PROJECT_DIR$" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_13" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (SteffiStuffel) [inline native/android/.idea/misc.xml:7]: Is it intended that language level is 13, but project-jdk-name is still 8.

And why have you updated this?

**Review by SteffiStuffel — APPROVED** [review]

File: native/android/.idea/misc.xml (hunk 2 above)
Comment (maxammann) [reply native/android/.idea/misc.xml:7]: This should either be JDK 8 or JDK 15. JDK 15 is the current version of java whereas jdk8 is the legacy version

**Review by maxammann — APPROVED** [review]

File: native/android/.idea/misc.xml (hunk 2 above)
Comment (Taggotty) [reply native/android/.idea/misc.xml]: will revert. IntelliJ updated it by itself, as I did have another jdk on my system

**Pull request merged**